holidayextras/make-up
# ESLint 2.8.0

#### What does this PR do? (please provide any background)
I've updated make-up to use the new eslint 2 ruleset from https://github.com/holidayextras/culture/pull/105
I've updated make-up to use eslint 2.2.0 and the updated react & mocha plugins.
I've had to fix some linting issues in the eslint integration...
I guess the consistent-return rule works now.

#### What tests does this PR have?
The existing tests.

#### How can this be tested?
Run this branch of make-up in tripapp or any other project that uses make-up. It should still work as normal.

#### Screenshots / Screencast


#### What gif best describes how you feel about this work?
![](http://i.imgur.com/B9fJh45.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
